build/Toolset/Toolset.csproj

```diff
@@ -10,7 +10,7 @@
     <RootNamespace>Toolset</RootNamespace>
     <AssemblyName>Toolset_DoNetUse</AssemblyName>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
-    <RuntimeIdentifiers>win7;ubuntu.14.04;osx.10.10</RuntimeIdentifiers>
+    <RuntimeIdentifiers>win7;rhel.7;ubuntu.14.04;osx.10.10</RuntimeIdentifiers>
```

Comment:
There are a few places where we don't include an architecture when specifying a RID.  What's the reason behind that @jaredpar or @agocke?